itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/CreateNewQuarkusConfigurationMavenTest.java

```diff
@@ -0,0 +1,121 @@
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.eclipse.condition.ConsoleHasText;
+import org.eclipse.reddeer.eclipse.debug.ui.launchConfigurations.LaunchConfiguration;
+import org.eclipse.reddeer.eclipse.ui.console.ConsoleView;
+import org.eclipse.reddeer.eclipse.ui.navigator.resources.ProjectExplorer;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenuItem;
+import org.eclipse.reddeer.swt.impl.table.DefaultTableItem;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.quarkus.reddeer.perspective.QuarkusPerspective;
+import org.jboss.tools.quarkus.reddeer.wizard.CodeProjectTypeWizardPage;
+import org.jboss.tools.quarkus.reddeer.wizard.QuarkusWizard;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * https://issues.redhat.com/browse/JBIDE-27065
+ * New quarkus launch configuration integration test for Maven
+ * @author olkornii@redhat.com
+ */
+@OpenPerspective(QuarkusPerspective.class)
+@RunWith(RedDeerSuite.class)
+public class CreateNewQuarkusConfigurationMavenTest extends LaunchConfiguration {
+
+	public CreateNewQuarkusConfigurationMavenTest() {
+		super("Create New Quarkus Configuration Maven Test");
+	}
+
+	private static String MAVEN_PROJECT_NAME = "testProjectMaven";
+	private static ConsoleView consoleView;
+
+	@BeforeClass
+	public static void testNewNewQuarkusMavenProject() {
+		new WorkbenchShell().setFocus();
+
+		QuarkusWizard qw = new QuarkusWizard();
+		qw.open();
+		assertTrue(qw.isOpen());
+
+		CodeProjectTypeWizardPage wp = new CodeProjectTypeWizardPage(qw);
+		wp.setProjectName(MAVEN_PROJECT_NAME);
+		wp.setMavenProjectType();
+
+		qw.next();
+		qw.finish(TimePeriod.VERY_LONG);
+
+		assertTrue(new ProjectExplorer().containsProject(MAVEN_PROJECT_NAME));
+
+		consoleView = new ConsoleView();
+	}
+
+	@Test
+	public void testNewQuarkusMavenConfiguration() {
+		CreateNewQuarkusMavenConfiguration();
+		CheckNewQuarkusMavenConfiguration();
+		RunNewQuarkusMavenConfiguration();
+	}
+
+	public void CreateNewQuarkusMavenConfiguration() {
+		new WorkbenchShell().setFocus();
+		new ProjectExplorer().selectProjects(MAVEN_PROJECT_NAME);
+
+		new ContextMenuItem("Run As", "Run Configurations...").select();
+		new WaitUntil(new ShellIsAvailable("Run Configurations"));
+
+		new DefaultTreeItem("Quarkus Application").select();
+		new ContextMenuItem("New Configuration").select();
+
+		setName(MAVEN_PROJECT_NAME + "Configuration");
+
+		new PushButton("Browse...").click();
+		new DefaultTableItem(MAVEN_PROJECT_NAME).select();
+		new PushButton("OK").click();
```

Comment:
There are some predefined buttons like OkButton, FinishButton, NextButton....you can use these if you want.
See: https://github.com/eclipse/reddeer/tree/37f27affd709a9622d84e8d29d047dc78a0de1b4/plugins/org.eclipse.reddeer.swt/src/org/eclipse/reddeer/swt/impl/button